client/src/gui/components/changelog.rs

```diff
@@ -1,7 +1,6 @@
 use crate::{consts, gui::views::default::DefaultViewMessage, net, Result};
 use iced::{scrollable, Element};
 use serde::{Deserialize, Serialize};
-
```

Comment:
was that intended?

Reply:
explictly by me, no.
implicitly by cargo fmt, yes.